app/model/entity/ExerciseData.php

```diff
@@ -0,0 +1,170 @@
+<?php
```

Comment:
We might want to create new traits in the near future. Consider making new namespace, or at least new folder which will hold traits. It might become a little bit messy, what is actual entity and what is not...

Reply:
That's a good point, I'll move it.